### PR TITLE
change service integration behaviour

### PR DESCRIPTION
**Pull request by ivan-savciuc**
In some cases, service creation may trigger the creation of service integration. For example, PG read replica service integration does that. PG replica service is created with service integration user configuration options that trigger the creation of service integration. To handle such cases correctly, we update the creation logic for the service_integration resource; it should work correctly when a service integration already exists.

### CODE REVIEW

**Review by encima — DISMISSED** [review]
Only tested by running tests existing. Check for adding integrations to already running services makes sense but would have liked to test. Approving now as I am holding up the release. Will merge by EOD unless more stringent checks are needed and then I will make time on Monday

**Comment by ivan-savciuc** [comment]
Updated this branch with the latest master changer

**Review by mhoffm-aiven — CHANGES_REQUESTED** [review]
purely cosmetical, but could we separate stdlib imports from external imports?

**Comment by ivan-savciuc** [comment]
> purely cosmetical, but could we separate stdlib imports from external imports?

sure, will do, I'll try to configure my IDE so it can do that automatically

**Review by mhoffm-aiven — APPROVED** [review]

**Pull request merged**
